vita3k/modules/SceAvPlayer/SceAvPlayer.cpp

```diff
@@ -39,108 +39,109 @@ constexpr bool CATCHUP_VIDEO_PLAYBACK = true;
 
 namespace emu {
-    typedef Ptr<void> (*SceAvPlayerAllocator)(void *arguments, uint32_t alignment, uint32_t size);
-    typedef void (*SceAvPlayerDeallocator)(void *arguments, void *memory);
-
-    typedef int (*SceAvPlayerOpenFile)(void *arguments, const char *filename);
-    typedef int (*SceAvPlayerCloseFile)(void *arguments);
-    typedef int (*SceAvPlayerReadFile)(void *arguments, uint8_t *buffer, uint64_t offset, uint32_t size);
-    typedef uint64_t (*SceAvPlayerGetFileSize)(void *arguments);
-
-    typedef void (*SceAvPlayerEventCallback)(void *arguments, int32_t event_id, int32_t source_id, void *event_data);
-
-    enum class DebugLevel {
-        NONE,
-        INFO,
-        WARNINGS,
-        ALL,
-    };
-
-    struct SceAvPlayerMemoryAllocator {
-        Ptr<void> user_data;
-
-        // All of these should be cast to SceAvPlayerAllocator or SceAvPlayerDeallocator types.
-        Ptr<void> general_allocator;
-        Ptr<void> general_deallocator;
-        Ptr<void> texture_allocator;
-        Ptr<void> texture_deallocator;
-    };
-
-    struct SceAvPlayerFileManager {
-        Ptr<void> user_data;
-
-        // Cast to SceAvPlayerOpenFile, SceAvPlayerCloseFile, SceAvPlayerReadFile and SceAvPlayerGetFileSize.
-        Ptr<void> open_file;
-        Ptr<void> close_file;
-        Ptr<void> read_file;
-        Ptr<void> file_size;
-    };
-
-    struct SceAvPlayerEventManager {
-        Ptr<void> user_data;
-
-        // Cast to SceAvPlayerEventCallback.
-        Ptr<void> event_callback;
-    };
-
-    struct SceAvPlayerInfo {
-        SceAvPlayerMemoryAllocator memory_allocator;
-        SceAvPlayerFileManager file_manager;
-        SceAvPlayerEventManager event_manager;
-        DebugLevel debug_level;
-        uint32_t base_priority;
-        int32_t frame_buffer_count;
-        int32_t auto_start;
-        uint8_t unknown0[3];
-        uint32_t unknown1;
-    };
-
-    struct SceAvPlayerAudio {
-        uint16_t channels;
-        uint16_t unknown;
-        uint32_t sample_rate;
-        uint32_t size;
-        char language[4];
-    };
-
-    struct SceAvPlayerVideo {
-        uint32_t width;
-        uint32_t height;
-        float aspect_ratio;
-        char language[4];
-    };
-
-    struct SceAvPlayerTextPosition {
-        uint16_t top;
-        uint16_t left;
-        uint16_t bottom;
-        uint16_t right;
-    };
-
-    struct SceAvPlayerTimedText {
-        char language[4];
-        uint16_t text_size;
-        uint16_t font_size;
-        emu::SceAvPlayerTextPosition position;
-    };
-
-    union SceAvPlayerStreamDetails {
-        SceAvPlayerAudio audio;
-        SceAvPlayerVideo video;
-        SceAvPlayerTimedText text;
-    };
-
-    struct SceAvPlayerFrameInfo {
-        Ptr<uint8_t> data;
-        uint32_t unknown;
-        uint64_t timestamp;
-        emu::SceAvPlayerStreamDetails stream_details;
-    };
-}
+typedef Ptr<void> (*SceAvPlayerAllocator)(void *arguments, uint32_t alignment, uint32_t size);
```

Comment:
put unrelated format changes like this (all of them below here) to another "all: Format project" commit